### DISCUSSION TITLE
cannot load local .bedpe into SV inspector

### DISCUSSION THREAD

**Discussion by charliechen912ilovbash** [GMOD/jbrowse-components#2073] [General] [answered]
Hi,

I tried importing the example  .vcf with url and it work,
but it seems does not work when importing files with "choosing files"
I downloaded the example .vcf and tried to import it with "choosing files" and failed.

**Comment by cmdcolin** [comment, marked as answer]
Thanks so much for reporting. I found that we had a bug in our 1.3.0 release related to this (we did a big overhaul to allow tracks to open local files which ended up breaking it in our sv inspector)

I made a proposed fix here
https://github.com/GMOD/jbrowse-components/pull/2074

**Reply by cmdcolin** [reply]
We should have a new release out with this fix somewhat soon but if you want to try you can use `jbrowse create --branch support_local_file_spreadsheet newfolder`

**Reply by charliechen912ilovbash** [reply]
thanks! It worked.
Looking forward to the new release!
